wagtail/admin/templates/wagtailadmin/pages/workflow_history/list.html

```diff
@@ -35,9 +35,9 @@ <h2>{{ workflow_name }}</h2>
                         {% for task in workflow_state.all_tasks_with_status %}
                             <span data-wagtail-tooltip="{{ task.name }}: {{ task.status_display }}">
                                 {% if task.status == 'approved' %}
-                                    {% icon "success" title=approved_title %}
+                                    {% icon "success" title=approved_title class_name="uniform" %}
                                 {% else %}
-                                    {% icon "radio-empty" title=incomplete_title %}
```

Comment:
Now handled in feature/workflow branch via 949d559cb81070623e36f8541827f85150a4f341.